aea/configurations/base.py

```diff
@@ -182,7 +183,7 @@ class ConnectionConfig(Configuration):
 
     def __init__(self,
                  name: str = "",
-                 authors: str = "",
+                 user_name: str = "",
```

Comment:
I would prefer `owner` or `owner_name` or smth. like that. To explain which user_name is that.

Reply:
We will switch back to author to avoid any conflict.